packages/backend-core/src/redis/redis.ts

```diff
@@ -111,6 +111,10 @@ function init(selectDb = DEFAULT_SELECT_DB) {
   CLIENTS[selectDb] = client
 }
 
+export function closeAll() {
+  Object.values(CLIENTS).forEach(client => client.disconnect())
+}
```

Comment:
I've added this because `redis.spec.ts` was giving me an error when the tests finished running saying there were open connections that could prevent Jest from finishing.

Reply:
I think we have been hunting this one for a while... 🤞🏽